tests/drivers/uart/uart_async_api/boards/cyw920829m2evk_02.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_TEST_USERSPACE=n
```

Comment:
Why are we disabling user space?

Reply:
It requires MPU region configuration. This will be added in https://github.com/zephyrproject-rtos/zephyr/pull/73748 (https://github.com/zephyrproject-rtos/zephyr/pull/73748/commits/95d3874c01964246a81b4cc727b529472d76e63b).

For UART test it is not critical, so disable so far...